src/utils/index.ts

```diff
@@ -638,7 +638,8 @@ export function isMobileDevice() {
   return device && device.type === 'mobile';
 }
 
-export const isSafari = () => parsedUserAgent.getBrowser().name === 'Safari';
+export const isSafari = () =>
+  parsedUserAgent.getBrowser().name?.toLowerCase().includes('safari') || false;
```

Comment:
add IOS check as well. Mac safari works fine. we don't need warning for that. use `!!` instead of `|| false`